crawl-ref/source/player-equip.cc

```diff
@@ -632,10 +632,9 @@ static void _unequip_weapon_effect(item_def& real_item, bool showMsgs,
 
     // Fragile artefacts may be destroyed, so make a copy
     item_def item = real_item;
-
-    // Call this first, so that the unrandart func can set showMsgs to
-    // false if it does its own message handling.
-    if (is_artefact(item))
+    // If item is an unrand, call this first, so that the unrandart func
```

Comment:
This variable is probably not even needed now, but I'll leave it here in case some Fragile unrand one day gets added.